react-props-and-expressions/my-app/src/App.js

```diff
@@ -0,0 +1,37 @@
+import './App.css';
+
+function CustomButtonClass({ text, color }) {
```

Comment:
The exercise suggested you put these into a separate file. That's the most common way to do reusable components. That way they can be easily reused in other places.